### DISCUSSION TITLE
More UI Customization

### DISCUSSION THREAD

**Discussion by Fudgecrumb1** [ScratchAddons/ScratchAddons#4963] [Ideas]
In my opinion, I think SA should add more customizability for Scratch's UI. I would love to see a feature that would allow users to make their UI look like this: https://scratch.mit.edu/projects/719334055/ if they wanted to.

The addons SA are already amazing, and I think this would make it much better, and possibly attract more users!

**Comment by DNin01** [comment]
Well, if you have any specific ideas to fine-tune Scratch to your liking, you can always [create some issues](https://github.com/ScratchAddons/ScratchAddons/issues/new/choose) to suggest your ideas.

Cool mockup, by the way!

**Reply by Hans5958** [reply]
Hey, you can also use GitHub Discussion for these kind of ideas also. It's fine.
